vault/expiration.go

```diff
@@ -1131,33 +1131,33 @@ func (m *ExpirationManager) expireID(leaseID string) {
 }
 
 // revokeEntry is used to attempt revocation of an internal entry
-func (m *ExpirationManager) revokeEntry(le *leaseEntry) error {
+func (m *ExpirationManager) revokeEntry(ctx context.Context, le *leaseEntry) error {
 	// Revocation of login tokens is special since we can by-pass the
 	// backend and directly interact with the token store
 	if le.Auth != nil {
-		if err := m.tokenStore.revokeTree(m.quitContext, le.ClientToken); err != nil {
+		if err := m.tokenStore.revokeTree(ctx, le.ClientToken); err != nil {
```

Comment:
Same deal, not sure we want to risk cutting this off partway through (unneccessarily).

Reply:
The active context is given by the caller of the public methods. The context that ends here from the caller of `Revoke` and `RevokeByToken` is the active context.